src/niswitch/metadata/config.py

```diff
@@ -0,0 +1,24 @@
+config = {
+    'metadata_version': '1.0',
+    'module_name': 'niswitch',
+    'module_version': '0.1.0.dev3',
+    'c_function_prefix': 'niSwitch_',
+    'driver_name': 'NI-SWITCH',
+    'session_description': 'An NI-SWITCH session to a National Instruments Switch Module',
+    'library_info':
+    {
+        'Windows': {
+            '32bit': {'name': 'niswitch_32.dll', 'type': 'windll'},
+            '64bit': {'name': 'niswitch_64.dll', 'type': 'cdll'},
+        },
+        'Linux': {
+            '64bit': {'name': 'libniswitch.so', 'type': 'cdll'},
+        },
+    },
+    'context_manager': [
+        {
+            'direction': 'input'
```

Comment:
@texasaggie97 I don't know if 'direction' will be the way to go for this. If I remember this right, we use this exclusively (today) to name the context manager for Acquisition vs Generation. I don't think either applies to NI-SWITCH, and NI-HSDIO would also be problematic.

@Fladolcetta made do a simple scanning example, to illustrate what this would look like (separate PR).

If there's agreement, I can open an Issue and not hold this PR.

Reply:
I agree this can be revisited to see if there is a better

Reply:
Opened https://github.com/ni/nimi-python/issues/267